<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lcModelListDialog</class>
 <widget class="QDialog" name="lcModelListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Submodels</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="ModelList">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="NewModel">
         <property name="text">
          <string>&amp;New...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DeleteModel">
         <property name="text">
          <string>D&amp;elete...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RenameModel">
         <property name="text">
          <string>&amp;Rename...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ExportModel">
         <property name="text">
          <string>Export...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DuplicateModel">
         <property name="text">
          <string>Duplicate...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="MoveUp">
         <property name="text">
          <string>Move &amp;Up</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="MoveDown">
         <property name="text">
          <string>Move &amp;Down</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="SetActiveModel">
         <property name="text">
          <string>Set Active</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ModelList</tabstop>
  <tabstop>NewModel</tabstop>
  <tabstop>DeleteModel</tabstop>
  <tabstop>RenameModel</tabstop>
  <tabstop>ExportModel</tabstop>
  <tabstop>DuplicateModel</tabstop>
  <tabstop>MoveUp</tabstop>
  <tabstop>MoveDown</tabstop>
  <tabstop>SetActiveModel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>lcModelListDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>lcModelListDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
